Package.swift

```diff
@@ -35,6 +35,10 @@ let package = Package(
             name: "BraintreePayPal",
             targets: ["BraintreePayPal", "PPRiskMagnes"]
         ),
+        .library(
+            name: "BraintreePayPalMessaging",
```

Comment:
Is now the time to create new modules without the `Braintree` prefix? Or do we want to drop all the prefixes in v7 instead? Either way is fine by me just wanted to call it out

Reply:
The Braintree prefix isn't an Obj-C thing and I believe will always be required on iOS. It's used because CocoaPods requires a namespace. ex: merchants using this module with CocoaPods would do `pod 'Braintree/PayPalMessaging'`. @scannillo to keep me honest.

Now the `BT` prefix on classes/enums/etc is another question entirely. I'm not sure if we want to wait for v7 for that change or make it sooner ([I asked that question yesterday here](https://github.com/braintree/braintree_ios/pull/1142#discussion_r1418029329))

Reply:
Yeh, it's really just for "parity" with CocoaPods naming which requires the `Braintree` prefix. It is up to us, though!